backend/src/parser/TheParser.ts

```diff
@@ -42,6 +43,10 @@ export default class TheParser {
   private readonly parserConfig: ParserConfig
   private readonly mediaHostingUrlOrigin: string
   private readonly mediaHostingUrlBunny: string
+  private blockTags: string[]
```

Comment:
In fact we should do that for all properties that currently assigned static values in constructor
```suggestion
  static #blockTags = new Set(['blockquote', 'expand', 'pre'])
```

Reply:
agreed on static, disagreed on Set

Reply:
Oki, let's convert to static and then good to go

Reply:
> Oki, let's convert to static and then good to go

I thought a bit more about it, and I don't think converting to static makes much sense:

1. `Parser` is already singleton in prod, so what are we trying to win?
2. I deliberately put initialization of all tag-related "config-like" properties in a single place:
  `this.disallowedTagNesting`, `this.blockTags`, `this.allowedTags`, so that contributors have a better chance to discover them and change, when needed. E.g. #402 missed updating `blockTags`.
3. extensibility: in the future we might want to modify behavior of a concrete parser instance based on constructor arguments